assets/Yves/default/app/components/customer-reorder-widget/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * This file is part of the Spryker Demoshop.
```

Comment:
@dereuromark hmm

Reply:
We do not yet have the sniffer running for JS files.